src/Symfony/Component/Messenger/Tests/Command/SetupTransportsCommandTest.php

```diff
@@ -0,0 +1,95 @@
+        $serviceLocator = $this->createMock(ServiceLocator::class);
+        // get method must be call twice and will return consecutively a setup-able transport and a non setup-able transport
+        $serviceLocator->expects($this->exactly(2))
+            ->method('get')
+            ->will($this->onConsecutiveCalls(
+                $this->createMock(SetupableTransportInterface::class),
+                $this->createMock(TransportInterface::class)
+            ));
+        $serviceLocator
+            ->method('has')
+            ->willReturn(true);
+
+        $command = new SetupTransportsCommand($serviceLocator, ['amqp', 'other_transport']);
+        $tester = new CommandTester($command);
+        $tester->execute([]);
+        $display = $tester->getDisplay();
+
+        $this->assertContains('The "amqp" transport was setup successfully.', $display);
+        $this->assertContains('The "other_transport" transport does not support setup.', $display);
+    }
+
+    public function testReceiverNameArgument()
+    {
+        // mock a service locator
+        /** @var MockObject|ServiceLocator $serviceLocator */
+        $serviceLocator = $this->createMock(ServiceLocator::class);
+        // get method must be call twice and will return consecutively a setup-able transport and a non setup-able transport
+        $serviceLocator->expects($this->exactly(1))
+            ->method('get')
+            ->will($this->onConsecutiveCalls(
+                $this->createMock(SetupableTransportInterface::class)
+            ));
+        $serviceLocator->expects($this->exactly(1))
+            ->method('has')
+            ->willReturn(true);
+
+        $command = new SetupTransportsCommand($serviceLocator, ['amqp', 'other_transport']);
+        $tester = new CommandTester($command);
+        $tester->execute(['transport' => 'amqp']);
+        $display = $tester->getDisplay();
+
+        $this->assertContains('The "amqp" transport was setup successfully.', $display);
+    }
+
+    /**
+     * @expectedException \RuntimeException
+     * @expectedExceptionMessage The "not_found" transport does not exist.
+     */
+    public function testReceiverNameArgumentNotFound()
+    {
+        // mock a service locator
+        /** @var MockObject|ServiceLocator $serviceLocator */
+        $serviceLocator = $this->createMock(ServiceLocator::class);
+        // get method must be call twice and will return consecutively a setup-able transport and a non setup-able transport
+        $serviceLocator->expects($this->exactly(0))
+            ->method('get');
+        $serviceLocator->expects($this->exactly(1))
+            ->method('has')
+            ->willReturn(false);
+
+        $command = new SetupTransportsCommand($serviceLocator, ['amqp', 'other_transport']);
+        $tester = new CommandTester($command);
+        $tester->execute(['transport' => 'not_found']);
+        $display = $tester->getDisplay();
```

Comment:
As `$tester->execute(...)` will throw an exception, everything below this is useless, can you remove them please?